.golangci.yml

```diff
@@ -43,4 +43,4 @@ linters:
 
 issues:
   # Only show newly introduced problems.
-  new-from-rev: 36838cf7f464cf73b0201798063b2caffeae4250
+  new-from-rev: "1ea58ad3d61b6d09e10e1075258d19d0060e1938" 
```

Comment:
This commit can be dropped now that we do full clone?

Reply:
indeed, done